src/Infrastructure/Data/CatalogContext.cs

```diff
@@ -74,7 +73,7 @@ private void ConfigureCatalogBrand(EntityTypeBuilder<CatalogBrand> builder)
             builder.HasKey(ci => ci.Id);
 
             builder.Property(ci => ci.Id)
-               .ForSqlServerUseSequenceHiLo("catalog_brand_hilo")
+               .UseHiLo("catalog_brand_hilo")
```

Comment:
Is the old method deprecated? What's the difference between this and the old method?

Reply:
Yes and no difference.  I think they just renamed it as far as I can tell from docs/recommendations.